frontend/public/components/operator-hub/operator-hub-items.tsx

```diff
@@ -77,32 +79,66 @@ const providerSort = provider => {
   return provider.value;
 };
 
+enum ProviderType {
+  RedHat = 'Red Hat',
+  Certificate = 'Certified',
+  Community = 'Community',
+  Custom = 'Custom',
+}
+
+enum InstalledState {
+  Installed = 'Installed',
+  NotInstalled = 'Not Installed',
+}
+
+enum CapabilityLevel {
+  BasicInstall = 'Basic Install',
+  SeamlessUpgrade = 'Seamless Upgrade',
+  FullLifecycle = 'Full Lifecycle',
+  DeepInsight = 'Deep Insight',
+}
+
 const providerTypeSort = provider => {
   switch (provider.value) {
-    case 'Red Hat':
+    case ProviderType.RedHat:
       return 0;
-    case 'Certified':
+    case ProviderType.Certificate:
       return 1;
-    case 'Community':
+    case ProviderType.Community:
       return 2;
-    case 'Custom':
+    case ProviderType.Custom:
       return 4;
     default:
       return 5;
```

Comment:
This isn't new, but all other values will be sorted randomly below. Do these enums include all possible values?

Reply:
I don't see any other categories locally. I'm not sure where to verify this.

Reply:
Don't worry about it in this PR since it's not new.